types/node/globals.d.ts

```diff
@@ -63,6 +63,7 @@ interface AbortSignal extends EventTarget {
      */
     readonly aborted: boolean;
     readonly reason: any;
+    onabort: null | ((this: AbortSignal, event: Event) => any);
```

Comment:
Looks good! But I think that `/ts4.8/globals.d.ts` should be updated too.

Reply:
I update the whole `borrowed` region, is that right?

Reply:
Yeah, looks good for me 👍